tests/lib/modem_trace/modem_trace_rtt/testcase.yaml

```diff
@@ -0,0 +1,11 @@
+tests:
+  lib.modem_trace_rtt:
+    platform_allow: qemu_cortex_m3
+    integration_platforms:
+      - qemu_cortex_m3
+    tags: modem_trace
+    extra_configs:
+      # The promptless symbol HAS_SEGGER_RTT is defined in the tests' Kconfig file.
+      - CONFIG_USE_SEGGER_RTT=y
```

Comment:
Now that you are having this:
https://github.com/nrfconnect/sdk-nrf/blob/222656f1fa614a59192ed5c77bc2fcc482c9e62f/tests/lib/modem_trace/modem_trace_rtt/Kconfig#L1-L3

you should be able to move this setting to `prj.conf`.

The main purpose of `extra_configs` is to adjust individual test cases where they cannot share the settings in `prj.conf`.

That doesn't seem to be the case here as there is only a single test case.

Reply:
actually it should be possible to remove because `NRF_MODEM_LIB_TRACE_MEDIUM_RTT` is seleting this setting.